### PR TITLE
Calling API update

### PR DESCRIPTION
**Pull request by kanchm**
Notable changes in:
API
- call_subscribetotone.md
- call_unmute.md
- onlinemeeting_get.md
- participant_invite.md
Resources
- all files

Concepts\images need to be published to CDN.

### CODE REVIEW

File: api-reference/beta/resources/calls-api-overview.md [hunk 1]
```diff
@@ -83,9 +83,17 @@ The Media SDK allows the bot to send and receive audio, video, and video-based s
...
+## Testing
+
+Bots can be tested locally using tunneling services like [Ngrok](https://ngrok.com) following some setup. See [testing](https://cdn.graph.office.net/prod/GraphDocuments/en-us/concepts/calling-testing.md) to learn more.
+
 ## Known issues
 
 The following are known issues with the calls and online meetings API:
 
-- Case mismatch in payloads - Payloads returned in notifications and responses contain incorrect capitalization of odata.types. They are returned in upper camel case instead of lower camel case.
+- Case mismatch in payloads - Payloads returned in notifications and responses contain incorrect capitalization of odata.types. They are returned in upper camel case instead of lower camel case. Enums sent in requests need to be in upper camel case.
```
Comment (Lauragra) [inline api-reference/beta/resources/calls-api-overview.md]: Please add the Known issues to the Known issues topic in a new H2 for "Calls and online meetings". You can then mention and link to this topic for details about the known issues.

File: api-reference/beta/resources/calls-api-overview.md [hunk 2]
```diff
@@ -83,9 +83,17 @@ The Media SDK allows the bot to send and receive audio, video, and video-based s
...
 - Navigation path `/applications/{id}` is not supported - Navigating through the global applications node to the application, even your own, is not allowed. Please use the `/app` navigation only.
+
+## See also
+
+[Calls and online meetings API samples](https://github.com/microsoftgraph/microsoft-graph-comms-samples/).
```
Comment (Lauragra) [inline api-reference/beta/resources/calls-api-overview.md]: FYI, the original link here was broken. To be clear, are the samples in the repo for communications scenarios more broadly, or calls specifically? Please clarify what you're linking to and make sure that the link text is accurate.

File: api-reference/beta/resources/chatinfo.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# chatInfo resource type
+
+> **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Information about a message in Teams
```
Comment (Lauragra) [inline api-reference/beta/resources/chatinfo.md]: Global: For branding, please always refer to the product name as "Microsoft Teams". Please start with an imperative and end with a period. For example, revise as "Provides information about a message in Microsoft Teams."

Comment (kanchm) [reply api-reference/beta/resources/chatinfo.md]: Fixed it with latest update.

File: api-reference/beta/resources/calls-api-overview.md (hunk 2 above)
Comment (kanchm) [reply api-reference/beta/resources/calls-api-overview.md]: These cover the samples for calling and online meetings.  Applied your change to our sources.

File: api-reference/beta/resources/calls-api-overview.md (hunk 1 above)
Comment (kanchm) [reply api-reference/beta/resources/calls-api-overview.md]: Fixed as per recommendation.

Comment (kanchm) [reply api-reference/beta/resources/calls-api-overview.md]: Resolved in latest PR.

**Comment by kanchm** [comment]
@Lauragra I have cleaned up the easy tasks. I would need help with refactoring the content. Anyone I can work with and get the editorial review?

**Comment by Lauragra** [comment]
@kanchm , can you please put the new concept topics into a separate PR so that we can handle them separately? I can do an editorial review. Ideally you or someone on your team would handle the refactoring of the content. However, you might need to publish the updates and the chatInfo resource sooner. Thanks.

**Comment by kanchm** [comment]
The following PR extracts out the API changes and leaves behind the concepts topics and related files.
#3331

**Pull request closed without merging**